### ISSUE TITLE
[API] should return LFS size, not pointer size for `GET /repos/{owner}/{repo}/contents`

### ISSUE THREAD

**Issue by so2liu** [go-gitea/gitea#27034]
### Description

Hope: get LFS size for `GET /repos/{owner}/{repo}/contents` API

Current: return a pointer size like '135'

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

No

### How are you running Gitea?

docker compose

### Database

None

**Comment by KN4CK3R** [comment]
A note: Github also does only return the pointer size and not the real file size.

**Comment by lunny** [comment]
Yeah, we can do more than GH. Since we have stored LFS size in the database, I think it's not expensive and possible.

**Comment by starrysky8** [comment]
Can an additional lfs_size field be included in the API response?
